projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -1022,15 +1022,16 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
      * Clears focus inside the virtualized container on small scroll swaps.
      */
     protected scrollFocus(node?: HTMLElement): void {
-        const activeElement = this.document.activeElement as HTMLElement;
+        const document = (node.getRootNode() as any);
+        const activeElement = document.activeElement as HTMLElement;
```

Comment:
if you use the previous cast you probably won't need this one.

Reply:
Resolves to `Element` so still needs this cast to have `blur`.